vendor/wp-now/src/start-server.ts

```diff
@@ -44,6 +44,34 @@ export async function startServer( options: WPNowOptions = {} ): Promise< WPNowS
 	app.use( addTrailingSlash( '/wp-admin' ) );
 	const port = options.port ?? ( await portFinder.getOpenPort() );
 	const { php, options: wpNowOptions } = await startWPNow( options );
+
+	// Middleware to check if auto-login should be executed
+	app.use( async ( req, res, next ) => {
+		if ( req.query[ 'playground-auto-login' ] === 'true' ) {
+			await php.requestHandler.request( { url: '/wp-login.php' } );
+			const response = await php.requestHandler.request( {
+				url: '/wp-login.php',
+				method: 'POST',
+				body: {
+					log: 'admin',
+					pwd: options.adminPassword,
+					rememberme: 'forever',
+				},
+			} );
```

Comment:
I was considering to add an early return if the autologin fails. For example if the user has changed the password, then I was expecting this code would override the cookies and logout the user, but on the contrary it works like a charm.

I also tried with manual logout, and then using any of the wp-admin buttons and I was redirected to the login screen without any URL parameter, which seems the best flow for the user to enter their credentials.

So no need to add any changes 👍 

In any case, If the user restarts the site, the password will be overwritten.